src/Microsoft.AspNetCore.Mvc.Razor.Extensions/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.Extensions.targets

```diff
@@ -4,23 +4,31 @@
     <!--
       MVC will generally want to add support for runtime compilation, but only for applications. 
     -->
-    <GenerateRazorAssemblyInfo Condition="'$(GenerateRazorAssemblyInfo)'=='' and '$(OutputType)'=='Exe'">true</GenerateRazorAssemblyInfo>
+    <GenerateRazorAssemblyInfo Condition="'$(GenerateRazorAssemblyInfo)'==''">true</GenerateRazorAssemblyInfo>
```

Comment:
@rynowak we need to be able to generate `RelatedAssemblyPartAttribute` on all types of assemblies. Missed that in the intiial PR. I repurposed the attribute and added an additional attribute to specifically mean hosting attributes. We could have an alternate naming scheme too.

Reply:
Sounds good.